src/main/java/cz/xtf/TestConfiguration.java

```diff
@@ -180,7 +180,7 @@ public static String masterUrl() {
 		if (StringUtils.isNotBlank(result)) {
 			result = "https://" + (result.equals(CDK_DOMAIN) ? CDK_IP : ("api." + result)) + ":8443";
 		} else {
-			result = get().readValue(MASTER_URL);
+			result = "https://" + "api." + get().readValue(MASTER_URL);
```

Comment:
MASTER_URL property should already return the final url.

Reply:
So if you want to test on online with token, you should set up test.properties with eg.:

xtf.config.master.url=https://api.openshift.domain.com
xtf.config.master.token=somerandomhash
xtf.config.master.namespace=myproject